services/QuillLMS/client/webpack.client.dev.base.config.js

```diff
@@ -8,8 +8,8 @@ const railsEnv = process.env.RAILS_ENV || process.env.NODE_ENV
 
 const goFanoutUrl = process.env.GOLANG_FANOUT_URL;
 const pusherKey = process.env.PUSHER_KEY;
-const defaultUrl = process.env.DEFAULT_URL;
-const cdnUrl = process.env.CDN_URL;
+const defaultUrl = process.env.DEFAULT_UR || 'http://localhost:3000'
```

Comment:
looks like you're missing the final letter of `DEFAULT_URL`

```suggestion
const defaultUrl = process.env.DEFAULT_URL || 'http://localhost:3000'
```

Reply:
Thank you.  Great eye on that one.